api.swagger.yaml

```diff
@@ -14298,6 +14316,21 @@ components:
                   type: string
                   example: 'https://dev.invotra.com/api/0.3/blogs/01234567-89ab-cdef-1234-56789abcdef0'
                   format: uri
+    list-list-content-schema-write:
```

Comment:
I can still see 'uri' on the POST/PUT endpoints for a list:

https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/BO-67.SWAGGER-5.11/api.swagger.yaml

Reply:
try https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/BO-67.SWAGGER-5.13/api.swagger.yaml